pdc-contact-browser.spec

```diff
@@ -7,13 +7,18 @@ License:        MIT
 URL:            https://github.com/product-definition-center/pdc-contact-browser
 Source0:        %{name}-%{version}.tar.gz
 BuildArch:      noarch
+BuildRequires:  npm
 
 %description
 Web UI for viewing pdc contact
 
 %prep
 %setup -q -n %{name}-%{version}
 
+%build
+npm install
```

Comment:
Actually this is a problem. The build will try to download dependencies from NPM here, and that will fail in Koji or Brew, as those run a restricted environment with no network access. How is the spec file supposed to be used?

Checking last build in Brew, it uses a tarball that was created (most likely) by running `npm run build` in the repo and packing up everything. The build does not really do any building, it only copies existing files to other locations.

Reply:
Perhaps I can use system package `nodejs-rimraf` (`rimraf` is needed to build assets). Though after installing the package I get an error.

```bash
sh: rimraf: command not found
```

Looks like it's not installed in `PATH` and can be invoked only using nodejs.

Reply:
Never mind, `rimraf` only does cleaning, `webpack` is required to create the assets.